styles.css

```
@@ -0,0 +1,255 @@
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+
+/* --------------- 4.2 Introduction ------------------- */
+
+/* ------------------------------- 5. Media Queries -------------------------------- */
+
+button {
+    display: inline-block;
+    padding: 20px 50px;
+    font-size: 12px;
+    cursor: pointer;
+    text-align: center;
+    text-decoration: none;
+    outline: none;
+    color: white;
+    background-color: #ED4385;
+    border: none;
+    border-radius: 15px;
+    box-shadow: 0 5px #999;
+}
+button:active {
+    background-color: #3e8e41;
+    box-shadow: 0 5px #666;
+    transform: translateY(4px);
+}
+
+h1,h2,h3{
+    font-family: "Roboto Light", sans-serif;}
+
+p{
+    font-family:'Merriweather', sans-serif;}
+
+header{
+    background-color: #4845E4;
+    padding: 10px;
+}
+        nav{
+            display: flex;
+            width: 96%;
+            flex-direction: row;
+            justify-content: flex-end;
+            align-items: center;
+            align-content: stretch;
+            color: white;}
+
+        ul{ flex-basis: 280px;
+            list-style-type: none;
+            display: inline-flex;
+            flex-direction: row;
+            justify-content: space-between;}
+
+        .div-one{
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            text-align: center;
+            color: white;
+            min-width: 500px;
+            padding: 50px 400px;
+        }
```

Vervolgens willen we zorgen dat deze div dezelfde breedte blijft behouden van zichzelf, maar wel netjes in het midden blijft. We willen dus eigenlijk af van de padding's en margin's aan de linker en rechterzijde, maar wél de breedte dicteren:

```suggestion
        .div-one{
            display: flex;
            flex-direction: column;
            align-items: center;
            text-align: center;
            color: white;
            max-width: 500px;
            padding: 50px 0px;
        }
```

Verder kun je deze tructjes dan óók toepassen op div-two, want daar geldt eigenlijk hetzelfde voor: hier wordt ook niet netjes alles gecentreerd als het venster smaller wordt gemaakt